### DISCUSSION TITLE
hide uploader ip

### DISCUSSION THREAD

**Discussion by OlevO1** [rejetto/hfs#714] [Q&A] [answered]
Yoo how can i hide the uploader ip? 

![image](https://github.com/user-attachments/assets/095b8d55-bcca-405e-b900-33f879c9cd2a)

**Comment by rejetto** [comment, marked as answer]
mmm, for the moment you can use style
#menu-prop-uploader { display:none }

but looking back at this feature, it seems not ideal showing ip to anyone.
i'm not sure if i should show the "uploader" only to the admin,
or show uploader username to anyone and the ip only to admin.

**Reply by OlevO1** [reply]
thank you for the answer
